app-config.yaml

```diff
@@ -126,6 +126,14 @@ scaffolder:
       token:
         $secret:
           env: GITLAB_ACCESS_TOKEN
+  azure:
+    api:
+      organization:
```

Comment:
Is the organization secret? Could this just be a straight up value instead

Reply:
I did it as a secret since in our case we have a separate organization for our test environment and then I want it to be dynamic. I'm not sure how the environments really work so it might be a better way to do this. I can do it as a value in the examples though.